app/models/authenticator/amazon.rb

```diff
@@ -141,5 +141,9 @@ def aws_connect(access_key_id, secret_access_key, service = :IAM, proxy_uri = ni
         :log_formatter     => Aws::Log::Formatter.new(Aws::Log::Formatter.default.pattern.chomp)
       )
     end
+
+    def normalize_username(username)
+      username
```

Comment:
This seems really strange.  I've looked at [the base code](https://github.com/ManageIQ/manageiq/blob/9291810e126cd34e5931bbf985be430a7018a422/app/models/authenticator/base.rb#L283-L285) and I understand why this is here.

It's just strange that the subclass has the "reset" the default behavior (i.e., doesn't downcase the username) because the base implementation changes the default behavior (i.e., downcases the username).

Oh well, just something to keep in mind if this comes back to bite.